isis/src/control/objs/BundleSolutionInfo/BundleSolutionInfo.cpp

```diff
@@ -997,21 +1067,30 @@ namespace Isis {
     // Pad each element in the table with the space for the longest image
     // path/name then padd it the length of the element + 1
     QString header("Measures                            RMS(pixels)");
+    if (m_statisticsResults->outputLidarData()) {
```

Comment:
This whole chunk I had to re-write from scratch. I tested it and it looks correct to me, but I should update the unit test to check for this.